### DISCUSSION TITLE
de/1-01-ai-machine-learning/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#51] [General]
# de/1-01-ai-machine-learning/

https://ai.lernos.org/de/1-01-ai-machine-learning/

**Comment by Marianne910** [comment]
falscher Bezug / falscher link: "Der Dartmouth Workshop ist die Geburtsstunde der Künstlichen Intelligenz als Fachgebiet."
korrekt: die "Dartmouth conference" (Die Dartmouth Conference gilt als Geburtsstunde der Künstlichen Intelligenz als Forschungsgebiet. vollständiger Name: Dartmouth Summer Research Project on Artificial Intelligence https://de.wikipedia.org/wiki/Dartmouth_Conference) 
<=>  " Dathmouth workshop" (the longest continuous bilateral dialogue between American and Soviet representatives.)

**Reply by simondueckert** [reply]
Referenz ist für uns im Leitfaden die englische Wikipedia (https://en.wikipedia.org/wiki/Dartmouth_workshop), deswegen "Dartmouth Workshop", denn die Konferenz war eine Friedenskonferenz.

**Reply by Marianne910** [reply]
ja im englischen Wikipedia heißt es "Workshop" im deutschen Wikipedia "Conference" - die Verlinkung zum englischen Wikipedia ist trotzdem falsch sie führt zur Friedens Conference  (einfach mal testen) - der korrekte link zur englischen Wikipedia wäre: https://en.wikipedia.org/wiki/Dartmouth_workshop

**Comment by rednaxelaesor** [comment]
1973 KI-Winter -> Besser:  Erste KI-Winter. Laut Wikipedi gab es 1987 den zweiten KI-Winter

**Comment by fdenzer** [comment]
Die Zeitlinie stimmt ja nachdem ich bei Wikipedia (englisch) nachgelesen habe, so ziemlich gar nicht.
Generative Werkzeuge: 19tes Jahrhundert.
Andere Technologien, die hier so von 1997 bis 2017 stehen, sinder auch als bis spätestens 1974 erfunden worden.
Was neu ist, ist mehr Rechenleistung. Und etwas skrupellose verwendung von Big-Data- und Clockworker-Ansätzen. Damit kommen jetzt zugegeben mehr "Produkte" auf den Markt, aber bisher muss sich zeigen, ob außer Übersetzungen und Rechtschreibkorrektur bei LLMs etwas Sinnvolleres als mit symbolischer KI möglich ist.

**Reply by fdenzer** [reply]
(Die Tabelle stimmt schon eher, mir ging es um die Grafik)

**Reply by simondueckert** [reply]
Uns ging es bei den Daten in der Tabelle weniger darum, wann die Grundlagen erfunden wurden, sondern wann das jeweilige Ereignis öffentlich sichtbar und für eine große Menge Personen zugänglich wurde. Welche Einträge zw. 1997-2017 meinst du konkret?

**Comment by KathaZ** [comment]
Abschnitt (A) 1 / Zeile (Z) 5, Vorschlag: Sprachassistenzsysteme (statt: Sprachassistenten)
A1/Z10: Infogra*f*ik
A1/Z11: der Künstliche*n* Intelligenz
A2/Z9: unter *dem* sich
A3/Z1: *ein* spezifischer Bereich,
A6, Titel: Large Language Models & Diffusion Models ODER Large-Language-Modelle und Diffusion-Modelle
A6/Z2: OpenAI)*,* 
Tabelle Meilensteine der KI: 
- manuell erstell*t*en Regeln
- Deep-Learning-Techniken
- Deep-Learning-Architektur
- weniger Trainingszeit benötigt als (Komma löschen)
- Tabelle ggf. ergänzen um ChatGPT-4 bzw. 2024 ChatGPT 4o mit neuen Features wie Verarbeitung von Bildern und Tönen

**Comment by Myszak64** [comment]
Typo im ersten Absatz: "Google Tanslate" --> "Google T**r**anslate".

**Comment by Myszak64** [comment]
**Was ist mit der "Turing-Maschine"?**
"Der britische Mathematiker Alan Turing entwickelte **1936** die Turingmaschine. Mit dieser Rechenmaschine konnte er beweisen, dass Maschinen die Fähigkeit besitzen, kognitive Prozesse auszuführen."

**Reply by simondueckert** [reply]
Hmm, die Turing Maschine war doch "nur" ein Rechensystem, das hatte glaube ich mit "Intelligenz" im heutigen Sinne der KI nicht viel zu tun.
